src/Elastic.Apm.AspNetCore/DiagnosticListener/AspNetCoreErrorDiagnosticsSubscriber.cs

```diff
@@ -4,18 +4,18 @@
 namespace Elastic.Apm.AspNetCore.DiagnosticListener
 {
 	/// <summary>
-	/// Activates the <see cref="AspNetCoreDiagnosticListener" /> which enables
+	/// Activates the <see cref="AspNetCoreErrorDiagnosticListener" /> which enables
 	/// capturing errors within an ASP.NET Core application.
 	/// </summary>
-	public class AspNetCoreDiagnosticsSubscriber : IDiagnosticsSubscriber
+	public class AspNetCoreErrorDiagnosticsSubscriber : IDiagnosticsSubscriber
```

Comment:
This is technically a breaking change but we could merge `AspNetCoreErrorDiagnosticsSubscriber` and `AspNetCorePageLoadDiagnosticListener` into a single one under the old name (`AspNetCoreDiagnosticsSubscriber`)  - that would not be a breaking change.